app/services/folio/locations.rb

```diff
@@ -13,6 +13,51 @@ def self.label(code:)
       instance.data.dig(code, 'discoveryDisplayName')
     end
 
+    # Output a CSV of all locations, flattening objects and converting UUIDs to codes
+    # @return [String] comma-separated data for all locations
+    def self.to_csv
+      headers = %w[institutionCode campusCode libraryCode primaryServicePointCode servicePointCodes]
+
+      data = instance.data.each_value.map do |location|
+        headers |= location.keys
```

Comment:
I wonder if we want to exclude the raw details, metadata, and UUIDs (except the location UUID which is probably useful for something), just to clean up the output?

Reply:
I had the same thought. Made some changes to do that